### ISSUE TITLE
>64 sample sizes returns results for 64 samples.

### ISSUE THREAD

**Issue by Stealeristaken** [amazon-science/chronos-forecasting#55]
Hi there! I recently utilized Chronos for a school project and encountered an issue regarding prediction length. Specifically, when setting the prediction length higher than 64 and using the sample size as the prediction length, Chronos consistently returns the same predictions, regardless of the specified length. Despite changing the prediction length, the predictions remain consistent at [64, 64, 64, 9].

<code>
    <pre>
        <code>
df = ali[['date', 'open']]
context = torch.tensor(df["open"]) 
prediction_length = 201  
forecast = pipeline.predict(
    context,
    prediction_length,
    num_samples= 201,
    temperature=1.0,
    limit_prediction_length=False,
    top_k=50,
    top_p=1.0,
)</code>
    </pre>
</code>

And the result is; 
<img width="347" alt="Ekran Resmi 2024-04-18 08 54 17" src="https://github.com/amazon-science/chronos-forecasting/assets/109769982/1a28bb70-e95a-4dde-a3e7-c326f4c3711c">

**Comment by abdulfatir** [comment]
Hi @Stealeristaken! Thanks for your interest.

To clarify, `prediction_length` is the number of future steps that will be sampled along the time axis and `num_samples` is the number of trajectories that will be sampled. 

Can you check the following?

```py
print(forecast.shape, type(forecast))
```

In general, when opening issues, it would be best if you share a working script in place of a screenshot.

**Comment by abdulfatir** [comment]
@Stealeristaken I am closing this due to inactivity. Please feel free to re-open if you're still facing this issue.
